red-model/src/main/java/com/masi/red/entity/CandidateAnswer.java

```diff
@@ -0,0 +1,46 @@
+package com.masi.red.entity;
+import com.masi.red.common.Language;
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+
+@Data
+@Entity
+@Table(name = "candidate_answer")
+public class CandidateAnswer {
+
+    @Id
+    @GeneratedValue(generator = "optimized-sequence")
+    @Column(name = "id", nullable = false, unique = true)
+    private int id;
+
+    @NotNull
+    @ManyToOne(cascade = {CascadeType.ALL})
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    @NotNull
+    @ManyToOne(cascade = {CascadeType.ALL})
+    @JoinColumn(name = "test_id", nullable = false)
+    private Test test;
+
+    @NotNull
+    @ManyToOne(cascade = {CascadeType.ALL})
+    @JoinColumn(name = "question_id", nullable = false)
+    private Question question;
+
+    @NotNull
+    @Column(name = "answer",nullable = false)
+    private String answer;
+
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    @Column(name = "language", nullable = false)
+    private Language language;
```

Comment:
Kandydat będzie podawał język, w którym wypełnia test? Nie może to być wydedukowane z języka pytania?